test/tests/unit/mongodb_srv_tests.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
```

Comment:
Double checking: this is just renaming `mongodb_srv_spec_tests.js`?

Reply:
it does that, but also split the non-spec test involved into its own `describe` separate from the spec tests, and fixes that test not to use a SRV host that returns an `authSource`

Reply:
nah, if we moved it to the authproviders later that would be fine. Can we file a ticket for it?